### DISCUSSION TITLE
Save html and postgis

### DISCUSSION THREAD

**Discussion by ADChP** [opengeos/leafmap#132] [Q&A] [answered]
Hi! I'm trying to save a map with data from postgis. But it results a blank map. I've checked the credentials to access the database and all looks right. So, I wonder if is it possible to save the map with postgis data. I appreciate some support in this topic.

**Comment by giswqs** [comment, marked as answer]
Did you use the ipyleaflet plotting backend? Try the folium plotting backend if ipyleaflet doesn't work. 

https://leafmap.org/notebooks/19_map_to_html/

**Reply by ADChP** [reply]
Yes, I was using the ipyleaflet backend. I tried the folium backend and works perfectly. Thank you!
